libs/expo/betterangels/src/lib/screens/AddEditClient/ContactInfo.tsx

```diff
@@ -47,65 +59,52 @@ export default function ContactInfo(props: IContactInfoProps) {
           overflow: 'hidden',
         }}
       >
-        <BasicInput
-          onDelete={() => setClient({ ...client, address: '' })}
-          label="Mailing Address"
+        <Input
           placeholder="Enter Mailing Address"
-          value={client.address || ''}
-          onChangeText={(e) => setClient({ ...client, address: e })}
+          label="Mailing Address"
+          name="address"
+          control={control}
         />
-        <BasicInput
-          onDelete={() => setClient({ ...client, phoneNumber: '' })}
-          keyboardType="phone-pad"
-          label="Phone Number"
+        <Input
           placeholder="Enter Phone Number"
-          value={client.phoneNumber || ''}
-          onChangeText={(e) => setClient({ ...client, phoneNumber: e })}
+          label="Phone Number"
+          name="phoneNumber"
+          control={control}
+          keyboardType="phone-pad"
         />
-        <BasicInput
-          keyboardType="email-address"
-          onDelete={() =>
-            setClient({
-              ...client,
-              user: {
-                ...client.user,
-                email: '',
-              },
-            })
-          }
-          label="Email"
+        <Input
+          required
           placeholder="Enter Email"
-          value={client.user.email || ''}
-          onChangeText={(e) =>
-            setClient({
-              ...client,
-              user: {
-                ...client.user,
-                email: e,
-              },
-            })
-          }
+          label="Email"
+          error={!!errors['user']?.email}
+          name="user.email"
+          control={control}
+          keyboardType="email-address"
+          rules={{
+            required: true,
```

Comment:
should email be required?
![image](https://github.com/BetterAngelsLA/monorepo/assets/10333208/3a2175d6-3069-4639-8cbd-02f71dee058e)

Reply:
backend requires email field for update

Reply:
ah, yes. Sorry. That's on me